README.md

```diff
@@ -22,11 +22,12 @@ rules:
   from: 127.0.0.1:6443
   to:
     - 192.168.1.19:6443
-    - 192.168.1.21:6443
+    - 192.168.1.1@192.168.1.21:6443
     - 192.168.1.20:6443
 
 - name: rpi-ssh
   from: 127.0.0.1:22222
+  algorithm: round-robin
```

Comment:
In practise a random distribution works well.

What issue were you seeing with the current random allocation?

Reply:
From reading between the lines, it seems like this is a guess and hasn't been confirmed with data.

That's not enough to convince me that the extra complexity is warranted. It's very unlikely that you would have i.e. 3 upstreams, 100 connections in flight, and some upstreams with no work.